utils/CleanDatabases.ps1

```diff
@@ -0,0 +1,26 @@
+$DatabaseUrl = "http://localhost:8083"
+$DatabasePath = "C:\RavenDBv3\Data"
+$WaitMinutes = 0
+$ignore = @( "system", "databases" )
+
+function TC-Write([string]$name) {
+	Write-Output "##teamcity[message text='$name']"
+}
+
+TC-Write "Running RavenDB Database Cleaner"
+TC-Write "RavenDB URL: $DatabaseUrl"
+TC-Write "RavenDB Data Path: $DatabasePath"
+
+$databases = Get-ChildItem $DatabasePath -Directory `
+| Where-Object { $ignore -notcontains $_.Name } `
+| Where-Object { $_.CreationTimeUtc -lt (([DateTime]::UtcNow).AddMinutes(-$WaitMinutes)) }
+
+
+foreach($database in $databases) {
+    TC-Write "Deleting $database"
+    $deleteUrl = "$($DatabaseUrl)/admin/databases/$([Uri]::EscapeDataString($database.Name))?hard-delete=true"
```

Comment:
What's problematic is when it doesn't work the first time. ;-) Actually it works a lot better for V4 but it's not realistic to backport those changes to support branches as help from the Core AT framework was needed.

So far, it DOES work quite well for the support branches, which makes babysitting a slew of Raven builds MUCH nicer.